what3words.dotnet.wrapper/what3words.dotnet.wrapper.csproj

```diff
@@ -29,7 +29,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Refit" Version="6.0.24" />
+    <PackageReference Include="Refit" Version="5.2.4" />
```

Comment:
Is this downgrading the .NET sdk version? If so, would it have any usage implications?

Reply:
No, this is just specific to the `Refit` http lib that we use under the hood. Unfortunately, 6.0.24 doesn't work well with the dotnet core version we use for building this (currently using 3.1.x), at least based on how we build this, this works perfectly well on Visual Studio running on a Windows machine however as it uses a much more updated version of MSBuild that `Refit v6.0.24` supports.